harness/tests/cli/test_cli.py

```diff
@@ -444,3 +446,71 @@ def test_colored_str_output(case: Case) -> None:
     stream = io.StringIO()
     render.render_colorized_json(case.input, stream, indent="  ")
     assert stream.getvalue() == case.output + "\n"
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="requires Python3.8 or higher")
+def test_dev_bindings_parameter_inspect() -> None:
+    from determined.cli.dev import bindings_sig, can_be_called_via_cli, is_supported_annotation
+
+    _, params = bindings_sig(bindings.get_GetExperiment)
```

Comment:
Though this isn't traditionally how we've tested the CLI, we're trying to in general [move more to unit tests](https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1351909582/Python+Testing). Please let me know if you'd like to chat about it at some point, and no worries if that point isn't anytime soon.

Reply:
yeah, this will cause test failures when this particular API changes which isn't good. it's just been a lazy replacement for building out a list of annotations. I'll switch it out.